class.jetpack-debugger.php

```diff
@@ -264,7 +269,12 @@ public static function jetpack_debug_display_handler() {
 					<div style="clear: both;"></div>
 				</form>
 			<?php endif; ?>
-			</div>
+		</div> <!-- contact-message, hidden by default. -->
+		<div id="toggle_debug_info"><a href="#">View Advaned Debug Results</a></div>
```

Comment:
Seems like there is a small typo with `Advaned`, and the whole title should be translatable.